File: 2 Semestre/Tópicos Especiais de Sistemas/Projeto/Academia/ExercicioTreino.cs
class ExercicioTreino
{
    private Exercicio _exercicio;
    public Exercicio Exercicio{
        get => _exercicio;
    }
    private int _repeticoes;
    public int Repeticoes {
        get => _repeticoes;
    }
    private int _series;
    public int Series {
        get => _series;
    }

    public ExercicioTreino(Exercicio _exercicio, int _repeticoes, int _series){
        this._exercicio = _exercicio;
        this._repeticoes = _repeticoes;
        this._series = _series;
    }
}

File: 2 Semestre/Desenvolvimento de Software/Projeto/Projeto-Java/src/entities/ExercicioTreino.java
package entities;
public class ExercicioTreino {
    private Exercicio exercicio;
    private int series;
    private int repeticoes;

    public ExercicioTreino(Exercicio exercicio, int series, int repeticoes) {
        this.exercicio = exercicio;
        this.series = series;
        this.repeticoes = repeticoes;
    }
    public Exercicio getExercicio() {
        return exercicio;
    }
    public void setExercicio(Exercicio exercicio) {
        this.exercicio = exercicio;
    }
    public int getSeries() {
        return series;
    }
    public void setSeries(int series) {
        this.series = series;
    }
    public int getRepeticoes() {
        return repeticoes;
    }
    public void setRepeticoes(int repeticoes) {
        this.repeticoes = repeticoes;
    }
    @Override
    public String toString(){
        return "> Exercício: " + this.exercicio.getNome() + " - Series: " + this.series + " - Repetições: " + this.repeticoes;
    }
}
